upf/upf_cli.c

```diff
@@ -174,6 +174,38 @@ VLIB_CLI_COMMAND (upf_pfcp_show_endpoint_command, static) =
 };
 /* *INDENT-ON* */
 
+static clib_error_t *
+upf_pfcp_policer_set_fn (vlib_main_t * vm,
+			 unformat_input_t * main_input,
+			 vlib_cli_command_t * cmd)
```

Comment:
I don't see a definition of CLI command that uses this func. Also, binapi would be nice